### PR TITLE
Improvement/wallet integration

### PR DESCRIPTION
**Pull request by JonathanLangton1**
- Connect Chia wallets via WalletConnect
- Mobile responsive header
- New UI design
- Improved accessibility

### CODE REVIEW

File: src/app/bridge/ChiaWalletManager/wallets/goby.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// Wallet 1 specific logic
+export async function connect(): Promise<string> {
+  await window.chia.request({ method: "connect" })
+  const puzzle_hash = window.chia.selectedAddress
+  const address = GreenWeb.util.address.puzzleHashToAddress(puzzle_hash, CHIA_NETWORK.prefix)
+  return address
+}
+
+export function disconnect(): void {
+  console.log("Disconnecting Wallet 1...")
```
Comment (Yakuhito) [inline src/app/bridge/ChiaWalletManager/wallets/goby.tsx]: not disconnecting

File: src/app/bridge/ChiaWalletManager/wallets/walletconnect.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+export async function connect(ispersistenceConnect: boolean, setWalletConnectUri: (uri: string) => void): Promise<string> {
+  await getSession(ispersistenceConnect, setWalletConnectUri)
+  const res = await getAddress() as { data?: string }
+  const address = res?.data
+  if (!address) throw new Error('Failed to get Chia wallet address [WalletConnect]')
+  return address
+}
+
+export function disconnect(): void {
+  console.log("Disconnecting Wallet 2...")
```
Comment (Yakuhito) [inline src/app/bridge/ChiaWalletManager/wallets/walletconnect.tsx]: not disconnecting

**Comment by Yakuhito** [comment]
Hey! As discussed:
 - warp.green Bridging Interface title on the top left
 - bottom of the page: change to 'powered by warp.green' & add some bottom padding/margin
 - bridging step zero: get rid of the address field
 - for each bridging step, make sure eth wallet is on the right chain (e.g., if they're bridging from Sepolia, make sure the wallet is not connected to Base Sepolia - request network change otherwise)
 - Chia connect wallet modal: try using smaller buttons, put 'Connect Chia Wallet' in center, find a better position for the 'X'

Not necessary to have all of them in this PR - put what's relevant here, and get the others on the new branch. Thanks! 🫡

**Comment by Yakuhito** [comment]
Also, let's get this thing centered
image

File: src/app/bridge/ChiaWalletManager/wallets/goby.tsx (hunk 1 above)
Comment (JonathanLangton1) [reply src/app/bridge/ChiaWalletManager/wallets/goby.tsx]: Goby doesn't expose a disconnect method

File: src/app/bridge/ChiaWalletManager/wallets/walletconnect.tsx (hunk 2 above)
Comment (JonathanLangton1) [reply src/app/bridge/ChiaWalletManager/wallets/walletconnect.tsx]: Fixed ded05e0c94645f89cdc692fec20de437dc86f002

**Comment by Yakuhito** [comment]
Closing this - will merge with the other changes.

**Pull request closed without merging**